Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -148,48 +148,48 @@ public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationT
         }
 
         /// <inheritdoc />
-        public IEnumerable<PackageVersionInfo> GetCompatibleVersions(
-            IEnumerable<PackageVersionInfo> availableVersions,
+        public IEnumerable<VersionInfo> GetCompatibleVersions(
+            IEnumerable<VersionInfo> availableVersions,
             Version minVersion = null,
-            PackageVersionClass classification = PackageVersionClass.Release)
+            ReleaseChannel releaseChannel = ReleaseChannel.Stable)
```

Comment:
is this channel configurable?

Reply:
I think we settled on separate repositories instead of the channel, so this might be getting removed before a merge.

Reply:
remove then?

Reply:
Yeah, this isn't on the table for the next release so I'll get to it after that.

Reply:
So we aren't going to include this into 10.5.0, right?